src/Form/EmbedButtonDeleteForm.php

```diff
@@ -30,20 +22,4 @@ public function getCancelUrl() {
     return new Url('embed_button.list');
```

Comment:
Thats beautiful, great work @berdir on that issue.

On Fri, Jan 30, 2015 at 6:30 PM, Sascha Grossenbacher <
notifications@github.com> wrote:

> In src/Form/EmbedButtonDeleteForm.php
> <https://github.com/drupal-media/entity_embed/pull/123#discussion_r23840573>
> :
>
> > @@ -30,20 +22,4 @@ public function getCancelUrl() {
> >      return new Url('embed_button.list');
>
> Yes, you can :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/drupal-media/entity_embed/pull/123/files#r23840573>.
>



-- 
Regrads,
Chandan Singh
http://chandansingh.net/